docs/public-api.asciidoc

```diff
@@ -304,6 +304,33 @@ NOTE: Return value of <<convenient-capture-span>> method overloads that accept T
 
 NOTE: Code samples above use `Elastic.Apm.Agent.Tracer.CurrentTransaction`. In production code you should make sure the `CurrentTransaction` is not `null`.
 
+[float]
+[[api-transaction-ensure-parent-id]]
+==== `EnsureParentId`
```

Comment:
Do we want to add the version where we introduce this? We do that for configs, not sure if it's needed here.

Reply:
If we're going to introduce version tags to the API documentation, we should do it for all methods, not just this one. It's probably better to do that in a separate PR if so.